### PR TITLE
test for Ambulance  module

### LINKED ISSUE #2057: Test cases for ambulance module
part of epic issue - #1372

### PR DESCRIPTION
**Pull request by praffq**
## Proposed Changes

- Added test for various endpoints in the external_result module

### Associated Issue

-  Fixes #2057 

## Merge Checklist

- [x] Tests added/fixed
- [x] Linting Complete

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Comment by praffq** [comment]
@aeswibon  Can you check this PR once, I've followed what you've done in your closed PR with very minor changes. 
Please let me know of anything that can be improved or fix.

File: care/facility/tests/test_ambulance_api.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        driver_id = res.json()["id"]
+
+        res = self.client.delete(
+            self.get_url(
+                entry_id=self.ambulance.id,
+                action="remove_driver",
+            ),
+            data={"driver_id": driver_id},
+        )
+        print(res.data)
```
Comment (rithviknishad) [inline care/facility/tests/test_ambulance_api.py]:
```suggestion
```

**Review by rithviknishad — APPROVED** [review]

File: care/facility/api/viewsets/ambulance.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def get_serializer_class(self):
 
     @extend_schema(tags=["ambulance"])
     @action(methods=["POST"], detail=True)
-    def add_driver(self, request):
+    def add_driver(self, request, pk=None):
```
Comment (sainak) [inline care/facility/api/viewsets/ambulance.py]:
```suggestion
    def add_driver(self, request, *args, **kwargs):
```

File: care/facility/api/viewsets/ambulance.py [hunk 3]
```diff
@@ -78,7 +78,7 @@ def add_driver(self, request):
 
     @extend_schema(tags=["ambulance"])
     @action(methods=["DELETE"], detail=True)
-    def remove_driver(self, request):
+    def remove_driver(self, request, pk=None):
```
Comment (sainak) [inline care/facility/api/viewsets/ambulance.py]:
```suggestion
    def remove_driver(self, request, *args, **kwargs):
```

**Review by sainak — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**